add standard tags for db and redis suggested by OTEL

asldevi opened the pull request:
add standard tags for db and redis suggested by [open telemetry](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#mysql) and for better readability of spans from Jaeger/HT 
<img width="894" alt="Screenshot 2021-02-11 at 3 14 24 PM" src="https://user-images.githubusercontent.com/8813/107621163-d7aabc00-6c7b-11eb-90ab-cdf1f3210592.png">
UI.

Commit:
add standard tags for db and redis suggested by [otel](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#mysql)

subintp commented on src/Trace/Integrations/Redis.php:
```diff
@@ -52,30 +52,44 @@ public static function load()
+                    'db.system' => 'redis',
+                    'db.connection_string' =>  $connection_str,
+                    'redis.connection' => $connection_str,
+                    'span.kind' => Span::KIND_CLIENT
```
Can we take these tags from constants module or file ?

asldevi replied:
this is the way the library has been @subintp.

Commit:
fix redis and database tags (#10)

1. remove non standard redis.connection tag, blocking redis to be recognized as backend
2. for mysql dsn without host, take host as an optional param to PDO load

ShivamYuvraj committed:
merged master
